be/src/exec/pipeline/olap_chunk_source.cpp

```diff
@@ -329,7 +329,7 @@ Status OlapChunkSource::buffer_next_batch_chunks_blocking(size_t batch_size, boo
     for (size_t i = 0; i < batch_size && !can_finish; ++i) {
         ChunkUniquePtr chunk(
                 ChunkHelper::new_chunk_pooled(_prj_iter->encoded_schema(), _runtime_state->chunk_size(), true));
-        _status = _read_chunk_from_storage(_runtime_state, chunk.get());
```

Comment:
this code should not been try catched now, it may use the expr framework and some code of storage layer